packages/node/example/multiSession/README.md

```diff
@@ -0,0 +1,23 @@
+# Demo: a server NodeJS app managing sessions
+
+This demo shows how you can run a NodeJS server with multiple sessions in parallel.
+Check out the code in `src/serverSideApp.ts` for more details.
+
+Note that until [this issue](https://github.com/solid/node-solid-server/issues/1533)
+is resolved, this will not work against NSS. This demo is only intended to work against a compliant
+OIDC implementation, such as Inrupt's ESS (e.g. deployed at `https://pod.inrupt.com`).
```

Comment:
Our writing guidelines state 'i.e.' and 'e.g.' should be followed by a comma. I'd never of heard of it until Kevin brought it up recently (and it still seems weird to me), but apparently it is more correct English.
```suggestion
OIDC implementation, such as Inrupt's ESS (e.g., deployed at `https://pod.inrupt.com`).
```

Reply:
Oh right, yeah I was told that rule in the past